test/VisionModel/alexnet.jl

```diff
@@ -0,0 +1,7 @@
+using Flux
+using Test
```

Comment:
You don't need `using Flux; using Test` in each file.

Reply:
So, after adding them to ``runtests.jl``, right?